flake.nix

```diff
@@ -103,10 +103,7 @@
                 emmake make -j''${NIX_BUILD_CORES}
               '';
               checkPhase = ''
-                cd test
-                echo '{"type":"module"}' > package.json
-                node --experimental-wasm-threads -e '(async () => {const module = await import("./manifold_test.js"); await module.default();})()'
```

Comment:
I guess this is causing the nix failure somehow?

Reply:
probably, will check